Software/Core/src/main/java/org/foi/emp/core/DAO/ModelPracenjaDAO.java

```diff
@@ -22,5 +24,20 @@
     public void brisanjeModelaPracenja(ModelPracenja... modelPracenja);
 
     @Query("SELECT * FROM ModelPracenja")
-    public List<Kolegij> dohvatiSveModelePracenja();
+    public List<ModelPracenja> dohvatiSveModelePracenja();
+
+    @Query("SELECT * FROM ElementiModelaPracenja enmp where enmp.modelPracenja=:modelPracenja")
+    public List<ElementiNaModeluPracenja> dohvati(int modelPracenja);
+
+    @Query("SELECT * FROM ElementModelaPracenja where id=:modelPracenja")
+    public ElementModelaPracenja DohvatiElementModelaPracenja(int modelPracenja);
```

Comment:
You're using lower camel case annotation for functions (example: `unosModelaPracenja(),` but here you're using upper camel case `DohvatiElementModelaPracenja()`, `SHIFT + F6` and change it on every occurence.